### ISSUE TITLE
math.pow crashes CUDA enabled omniscidb server

### ISSUE THREAD

**Issue by pearu** [xnd-project/rbc#158]
Failure:
```
pytest -sv rbc/tests/test_omnisci_math.py -k 'test_math_function[pow]'

server:
LLVM ERROR: Cannot select: 0x7fe450066318: f64 = fpow 0x7fe4521a0050, 0x7fe4501a1dd0
  0x7fe4521a0050: f64,ch = load<(dereferenceable invariant load 8 from `double addrspace(101)* null`, addrspace 101)> 0x7fe450038c28, TargetExternalSymbol:i64'pow__0_param_0', undef:i64
    0x7fe4501a1958: i64 = TargetExternalSymbol'pow__0_param_0'
    0x7fe4501a2658: i64 = undef
  0x7fe4501a1dd0: f64,ch = load<(dereferenceable invariant load 8 from `double addrspace(101)* null`, addrspace 101)> 0x7fe450038c28, TargetExternalSymbol:i64'pow__0_param_1', undef:i64
    0x7fe4501a1f08: i64 = TargetExternalSymbol'pow__0_param_1'
    0x7fe4501a2658: i64 = undef
In function: pow__0
```

**Comment by guilhermeleobas** [comment]
After #198, the new error message is the one below

```
pytest -sv rbc/tests/test_omnisci_math.py -k 'test_math_function[pow]'

server:
2020-10-19T18:06:49.123708 F 23569 12 NativeCodegen.cpp:1057 use of undefined value '@__internal_accurate_pow'
Aborted (core dumped)
```

**Comment by pearu** [comment]
Resolved via #209
